index.css

```diff
@@ -21,8 +19,10 @@
     margin-left: 40px;
     margin-right: 40px;
     margin-top: 20px;
```

Comment:
Hvis dere bruker kun `margin-bottom`, så får dere litt mer kontroll over mellomrommet, i tillegg blir dere kvitt denne: 
![bilde](https://github.com/navikt/veilarbdetaljerfs/assets/85442404/b0adf9d3-cfdf-4678-b8fa-9996fb38d7e2)

Reply:
I tillegg så ville jeg heller lagt padding på `.main_grid` fremfor å ha `margin-left/-right`, også for å få mer kontroll!

Reply:
ja skjønner! takk, fikset nå!